app/controllers/general_shopping_list_controller.rb

```diff
@@ -0,0 +1,19 @@
+class GeneralShoppingListController < ActionController::Base
+  include GeneralShoppingListHelper
+
+  layout 'application'
+
+  def index
+    @ingredients = getting_shopping
+    @total_value = 0
+    @total_ingredients = 0
+    @ingredients.each do |i|
+      @total_value += Food.find(i.food_id).price * i.quantity
```

Comment:
You're running an extra query to the database here. `Food.find`

I believe this is considered a bad practice, but I don't know how to do it otherwise heheh 😴
It's fine for now.